Project: Building with Responsive Design 1st submition for review

jubaan opened the pull request:
Our approach was wider screens first and last mobile. We manage to set up the layout and the changes required for each one of the breakpoints. We found that TNW only uses 2 breakpoints smaller<768-1024<wider. We didn't put that much attention on the details, because the main goal was the layout and the use of media queries.

certilremy committed:
move the css file into a folder create a new branch for the features create the htlm element for pages

certilremy committed:
added the top and the main nav with style

jubaan committed:
FEAT: Cover grid 3 articles 1 big 2 small containers

Add the cover section where the main articles are displayed

jubaan committed:
ADD linear gradient to cover pictures

jubaan committed:
Add vendor prefix

jubaan committed:
erase vendero prefix and change text aling in article-big to center tnw logo

jubaan committed:
Fix Logo image is now top left of the cover container

certilremy committed:
move the main css to style.css , added the latest news section

jubaan committed:
FEAT: Sections of latest funding and latest deals documentation

Use latest news grid with modifications to reuse code and make the code
DRY

certilremy committed:
added big size sction

jubaan committed:
FEAT: all main containers are responsive except for the footer

All main containers do what they suppose to, the nav and the footer needs
more work, but almost done.

- Nav NOT needs more work
- Latest News DONE.
- Latest FUnding DONE.
- Latest Sections DONE.
- Latest Deals DONE.
- Footer NOT needs more work

jubaan committed:
FEAT: Navbar done with all responsive layout transitions

The navbar changes accordin to the layout of each meadia query it hiddes
also all the unnecesary elements.

ONLY thing left to do is insert the hamburger button to it.

jubaan committed:
FEAT: footer with al social media and links added, only details left

Footer is done:
- Social media icons
- links
- Legal info

jubaan committed:
DOC: fix all stylint issues and html validator issues

Check the file agains stylelinter and w3c html validator. No error left.

jubaan committed:
changes to the cover logo to move it right top

jubaan committed:
add max-with to asticle-big-photo to correct alignment

jubaan committed:
ADD img folder with original and solution images, change introduction and add images in README.

Anna-Myzukina commented:
### Pull Request under the review by Anna Muzykina (TSE)

Anna-Myzukina commented:
### Hi team!

### Your project looks good!  :+1: 
#### But there are a few things that need to be corrected:

#### Page structure

##### Navigation bar in Small version changes:
* Navigation and sub-navigation row should change to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/))

#### Original website in small version contains a hamburger icon
![original](https://user-images.githubusercontent.com/26605804/71126758-4f08cb00-21f2-11ea-88ca-b0dfe2d3c877.png)

#### Your clone in small version is without hamburger icon, kindly recommended to add it.
![nav](https://user-images.githubusercontent.com/26605804/71126763-529c5200-21f2-11ea-97b1-c592cf5a13fe.png)


#### CSS styles should use media queries for 3 breakpoints (screenshot from Pathwright)
![screen](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png)

* As I can see in your project just for 2 breakpoints @media only screen and (max-width: 768px) and 
@media only screen and (max-width: 1024px). Kindly recommended to add else one.


#### Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)


* Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

Anna-Myzukina commented on css/style.css:
```diff
@@ -0,0 +1,799 @@
+
+.article-big .article-section {
+  color: white;
+  font-size: 0.87 em;
```
Line number 290 : invalid property value 
Kindly recommended to delete space (write 0.87em instead 0.87 em)

Anna-Myzukina commented on css/style.css:
```diff
@@ -0,0 +1,799 @@
+
+.article-small .article-section {
+  color: white;
+  font-size: 0.87 em;
```
Line number 321 : invalid property value 
Kindly recommended to delete space (write 0.87em instead 0.87 em)

Anna-Myzukina commented on css/style.css:
```diff
@@ -0,0 +1,799 @@
+
+.section-details .first-article {
+  background: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/12/JBL-Reflect-Eternal-hed-368x276.jpg);
+  background-size: 100% 100% cover;
```
Line number 455 here is invalid property value, kindly recommended to fix it.

Anna-Myzukina commented on css/style.css:
```diff
@@ -0,0 +1,799 @@
+
+  .article-big .article-section {
+    color: white;
+    font-size: 0.87 em;
```
Line number 731 : invalid property value
Kindly recommended to delete space (write 0.87em instead 0.87 em)

Anna-Myzukina commented on index.html:
```diff
@@ -0,0 +1,1059 @@
+      <div class="latest-news-headline">
+        <h2>
+          <a href="javascript:void(0)">
+            Latest news >
```
Line number 120: Special characters must be escaped : [ > ]. Kindly recommended to delete symbol > after "Latest news"

Anna-Myzukina reviewed (changes requested):
The project needs changes

jubaan committed:
FIX: Page structure fixed, CSS style fixed DELETE: inline comments

Page structure issues:
- Add hamburger menu

CSS style:
- Add one more querie
- Delete inline comments
- Delete innecesary white spaces between values and units

nikasulo reviewed (changes requested):
# Nice job @jubaan and @certilremy! :rocket: 
There are a few things that need your attention before this project can be approved :smile: 

#### Horizontal Scroll
Your CSS is a little buggy, it has some horizontal scrolling going on.

#### Wrongly Sized Elements
A lot of the elements on the page are wrongly sized, please take a close look at the original size and adjust each element appropriately.

#### Missing Elements
You have a few things wrong with your navbar, elements are missing and it isn't the most responsive. Please look into this.

### Summary
You have done a nice job getting this far! Please take a closer look at your site's elements and responsiveness. Also, take a look at each element's design and try to adjust each appropriately! When all this is done, please request a re-review :+1:

Please reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin) if you think I missed something or you need some help!

certilremy committed:
fixed scroling error , fixed image bug on smalest screen fixed footer link

jubaan committed:
Fix lint and html validator issues

nikasulo reviewed (changes requested):
# Great work @jubaan and @certilremy!
You've implemented most of the changes beautifully, and there is just one more change you need to make. The section just beyond the latest funding section needs a bit of work :+1:, please look closely at the original and make the appropriate changes.

When you're done with that, please request a re-review!

certilremy commented:
Hello Tolulope Oluwadamilare Olusakin Thanks for your feedback . Can you please send us a capture or tell us briefly the chages that we need to do ?

certilremy commented:
> # Great work @jubaan and @certilremy!
> You've implemented most of the changes beautifully, and there is just one more change you need to make. The section just beyond the latest funding section needs a bit of work 👍, please look closely at the original and make the appropriate changes.
> 
> When you're done with that, please request a re-review!

Hello Tolulope Oluwadamilare Olusakin Thanks for your feedback . Can you please send us a capture or tell us briefly the chages that we need to do ?

nikasulo commented:
Thanks for reaching out @certilremy ! I will send a screenshot on slack.

certilremy commented:
> Thanks for reaching out @certilremy ! I will send a screenshot on slack.

Hello . Dont forget us , we are waiting.

shshamim63 commented:
## Status :tada:  [!! Approved !!] :tada: 
___
Hi @certilremy  and @jubaan :wave: .
Awesome work. No lint error, no HTML markup validation error. Code looks neat and clean. Readme is descriptive enough to guide me about this project. PR contain all the specific information about tasks that are covered in this PR. You have followed all the task provided by @Anna-Myzukina. Nothing is blocking me to declare this project as complete.
___
##### This :+1: is from me. You can merge it. Don't forget to fill the project completion form.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](__Shakhawat Hossain__)
